recipes/libarchive/all/conanfile.py

```diff
@@ -58,6 +58,14 @@ def source(self):
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    def validate(self):
```

Comment:
I can't remember about `validate` is it a new feature?

Reply:
yes, you can check it here https://blog.conan.io/2020/12/14/New-conan-release-1-32.html

Reply:
I would add `required_conan_version = ">=1.32.0"`

Reply:
@uilianries may I ask you to add this required_conan_version to the top?